data/schema.xsd

```diff
@@ -0,0 +1,968 @@
+<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" xmlns:xs="http://www.w3.org/2001/XMLSchema">
```

Comment:
Is this full scheme for check html as xml? I think it's very powerful solution little url_shortener. Maybe use something lite? For example: `https://docs.python.org/3/library/html.parser.html`

Reply:
I would prefer to use lxml, but assert not by XSD - which is really overhead - but using XPath for particular element on the page to ensure that we get expected page.